queries.yaml

```diff
@@ -985,6 +985,42 @@ post_process:
       properties:
         - walking_network
         - walking_shield_text
+  # drop RCN network & shield text below zoom 13.
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 13
+      properties:
+        - bicycle_network
+        - bicycle_shield_text
+      where: >-
+        bicycle_network == 'rcn'
+  # drop LCN network & shield text below zoom 16. this is earlier than a
+  # previous drop of the all_*, so make sure we clear those out too.
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - bicycle_network
+        - bicycle_shield_text
+        - all_bicycle_networks
```

Comment:
Dropping `all_bicycle_networks` and `all_bicycle_shield_texts` all the time until the max zoom would be best (and not limited to just `lcn`).